ui/webui/src/app.jsx

```diff
@@ -71,13 +118,16 @@ export const Application = () => {
                     );
                 })}
             </AlertGroup>}
-            <AddressContext.Provider value={address}>
-                {!path.length > 0 && <InstallationLanguage />}
-            </AddressContext.Provider>
-            {path.length > 0 &&
-            <AddressContext.Provider value={address}>
-                <Summary onAddNotification={onAddNotification} />
-            </AddressContext.Provider>}
+            <Wizard
```

Comment:
Since you have the info from the URL, for which step you are at, aka http://127.0.0.2:9091/cockpit/@localhost/anaconda-webui/index.html#/installation-destination for storage etc.

You can set the correct active step from the URL. Maybe check `activeStep` property? Did not try it out.